src/components/FilterPanel.jsx

```diff
@@ -73,9 +93,43 @@ export default function FilterPanel({ onClose }) {
             <Typography>{name}</Typography>
           </AccordionSummary>
 
-          <AccordionDetails sx={{ m: 1 }}>{filter}</AccordionDetails>
+          <AccordionDetails sx={{ m: 1 }}>{filter.comp}</AccordionDetails>
         </Accordion>
       ))}
+
+      {filtersChanged && (
+        <Box
+          sx={{
+            display: 'flex',
+            alignItems: 'center',
+            justifyContent: 'center',
+            mt: 2,
+          }}>
+          <WarningAmberOutlinedIcon color="warning" />
+          <Typography>Your changes have not yet been applied</Typography>
+        </Box>
+      )}
+
+      <Toolbar sx={{ justifyContent: 'space-evenly', mt: 2 }}>
```

Comment:
Why was a Toolbar chosen?

Reply:
What do you think of using a **Stack** instead and not centering the buttons and text but having them on the left? We could also add a margin of 2*spacing between the buttons so they're not so close to each other.

Reply:
Made the change but don't you think it looks off?

Reply:
Looks good to me?